build/Cortex-M3_MPS2_QEMU_GCC/target-specific-source/temp_mbedtls_freertos_port.c

```diff
@@ -241,10 +242,15 @@ int mbedtls_platform_entropy_poll( void * data,
     /* Context is not used by this function. */
     ( void ) data;
 
-    /* TLS requires a secure random number generator; use the RNG provided
-     * by Windows. This function MUST be re-implemented for other platforms. */
-    rngStatus = rand();//_RB_ Replace.
-//_RB_ Windows function        BCryptGenRandom( NULL, output, len, BCRYPT_USE_SYSTEM_PREFERRED_RNG );
+    /* Get entropy from rand(). Replace this with hardware TRNG on your platform */ 
+    for (size_t i = 0; i < len; i++) 
+    {
+        output[i] = (unsigned char) (rand() & 0xFF);
+    }
```

Comment:
These should match the rest of the style, namely spaces after parentheses/brackets. Also, I don't think size_t  is necessary
```suggestion
    for( int i = 0; i < len; i++ ) 
    {
        output[ i ] = ( unsigned char ) ( rand() & 0xFF );
    }
```